src/providers/wallet/mocks/wallet.mock.ts

```diff
@@ -210,15 +210,16 @@ export class WalletMock {
   }
   createTxProposal(_txp, cb) {
     const txp: TransactionProposal = {
+      coin: 'btc',
```

Comment:
createTx in BWS the `tx.coin` property comes from `opts.coin` from Copay to set `Chain` and `Currency` for PayProV2 requests.